content/pt-br/docs/concepts/overview/working-with-objects/kubernetes-objects.md

```diff
@@ -0,0 +1,90 @@
+content_type: concept
+weight: 10
+card:
+ name: concepts
+ weight: 40
+---
+ 
+<!-- overview -->
+Esta página explica como os objetos do Kubernetes são representados na API do Kubernetes e como você pode expressá-los no formato `.yaml`.
+ 
+<!-- body -->
+## Entendendo os objetos do Kubernetes  {#kubernetes-objects}
+ 
+*Os objetos do Kubernetes* são entidades persistentes no sistema do Kubernetes. Usuários do Kubernetes essas entidades para representar o estado do cluster. Especificamente, eles podem descrever:
+ 
+* Quais aplicativos estão sendo executados (e em quais nós).
+* Os recursos disponíveis para esses aplicativos
+* As políticas acerca de como esses aplicativos se comportam, como políticas de reinicialização e tolerâncias a falhas.
+
+ 
+Um objeto do Kubernetes é um “registro de intenção”--uma vez criado o objeto, o sistema do Kubernetes trabalha constantemente para garantir que este objeto existe. Ao criar um objeto, você está efetivamente falando para o sistema do Kubernetes como você quer que a carga do seu cluster seja. Este é o *estado desejado* do seu cluster.
+ 
+Para trabalhar com objetos do Kubernetes seja para criar, modificar ou deletar eles, você precisará usar a [Kubernetes API](/docs/concepts/overview/kubernetes-api/). Quando você usa a interface de linha de comando do  `kubectl`, por exemplo, o CLI faz as chamadas necessárias da API do Kubernetes para você. Você também pode usar a API do Kubernetes diretamente no seu próprio programa usando um dos [Client Libraries](/docs/reference/using-api/client-libraries/).
+ 
+
+
+### Especificação e status do objeto
+ 
+Quase todos os objetos do Kubernetes incluem dois campos de objetos aninhados que governam a configuração do objeto: a *`especificação`* do objeto e o *`status`* do objeto. Para objetos que têm especificação, você tem que definir isso quando você cria o objeto, fornecendo uma descrição das características que você quer que o recurso tenha: o seu _estado desejado_.
+ 
+O `status` descreve o _estado atual_ do objeto,  fornecido e atualizado pelo o sistema do Kubernetes e os componentes. O plano de controle do Kubernetes {{< glossary_tooltip text="control plane" term_id="control-plane" >}} gerência continuamente e ativamente o real estado para corresponder ao estado desejado que você forneceu.
+ 
+ 
+Por exemplo, no Kubernetes, o Deployment é um objeto que pode representar uma aplicação executando no seu cluster. Quando você cria o Deployment, você pode definir a `especificação` para especificar que você quer três réplicas da aplicação estejam rodando. O sistema de Kubernetes lê as especificações do Deployment e inicia três instâncias do seu aplicativo desejado, atualizando o status  para corresponder às suas especificações. Se uma dessas instâncias falhar (um status mudar), o sistema do Kubernetes responde as diferenças entre as especificações e o status fazendo uma correção--neste caso, iniciando uma instância de substituição.
```

Comment:
* can you use the glossary here for Deployment? I think it would be good to have the reference to the glossary
* The phrase "definir a especificação para especificar" is redundant. Maybe "alterar a `especificação` para definir que você quer três réplicas da aplicação em execução simultânea".
* "O sistema de Kubernetes" is a strange phrase IMO. maybe only "O Kubernetes lê as especificações do Deployment"

Let's not use "sistema de/do Kubernetes in any place and stick just with Kubernetes.

* Remove one of the dashes between correção--neste caso

Reply:
The Deployment glossary is in English, I didn't understand how to translate it, can you give me some hints?